Gems/GraphModel/Code/Source/Model/Slot.cpp

```diff
@@ -316,12 +316,17 @@ namespace GraphModel
         {
             AZStd::any slotValue;
             if (LoadAny<bool>(slotValue, serializedSlotValue->value, context, result) ||
-                LoadAny<int>(slotValue, serializedSlotValue->value, context, result) ||
+                LoadAny<int32_t>(slotValue, serializedSlotValue->value, context, result) ||
+                LoadAny<int64_t>(slotValue, serializedSlotValue->value, context, result) ||
+                LoadAny<uint32_t>(slotValue, serializedSlotValue->value, context, result) ||
+                LoadAny<uint64_t>(slotValue, serializedSlotValue->value, context, result) ||
```

Comment:
suggestion: Is it worth adding `int16_t` and `int8_t` (+ unsigned versions) while we're here too? 😛